components/monitoring/network/network.js

```diff
@@ -117,13 +127,37 @@ function activateNetwork() {
 
 function refreshNetwork() {
     updateNetworkUsage();
+    checkConnectivity();
+}
+
+function checkConnectivity(){
+    si.inetChecksite('https://google.com')
+	.then(data => {
+        console.log(data);
+		if (data.ms == -1) {
+            $("#internet-connected").hide();
+            $("#internet-disconnected").show();
+        } else {
+            $("#internet-connected").show();
+            $("#internet-disconnected").hide();
+        }
+	})
+	.catch(error => console.error(error));
 }
 
-function startPing() {
-    //TODO
-    // si.inetLatency((data) => {
-    //     console.log(data);
-    // });
+function doAPing() {
+    $("#ping-well").show();
+    $("#clearButton").show();
+    $("#pingButton").prop('disabled', true);
+    for (var i = 0; i < 4; i++) {
+        sleep(1000).then(() => {
+            si.inetLatency((data) => {
```

Comment:
This function seems to connect to 8.8.8.8 by default.
It would be nice to allow the user to change this to a custom IP address using an input field.
The placeholder attribute for the custom IP field could be set to `IP Address (8.8.8.8 by default)` or similar.

Unlike the checkConnectivity problem, this is not a blocking issue for me,
as it is up to the user to initiate the connection and it's not called each interval.